### ISSUE TITLE
Update the example code with the translation function

### ISSUE THREAD

**Issue by carolinan** [wp-bootstrap/wp-bootstrap-navwalker#476]
The example code for the menu is missing translation functions (and escaping, since this is inside an attribute) for `aria-label="Toggle navigation"`

#### What I expected:
I expect all text strings to be translation ready.

#### What happened instead:
After years of trying to track down the root of this issue that has **haunted** me for years, I finally found it, the first time(!!) I was tasked with implementing the nav walker myself.

**Meaning:**

The theme author copied the example code from this repository, 
and did not remember to add the translation function.

So what happened was that:
1) @pattonwebz had to tell all theme authors who submitted themes to 
the WordPress.org theme directory to update this text string manually, while he was doing the review.

2) @pattonwebz was pinged in **every theme** uploaded to WordPress.org that used the bootstrap navwalker, even if he was not the reviewer,  to tell the theme author to update the string manually, 

3) @pattonwebz had to create a check specifically for this problem in the Theme Check plugin.

4) @pattonwebz was from now on only known as "**Bootztrap Bill**".

**This is of course all speculation** about what may happen if @pattonwebz is not able to solve this issue.
